app/routes/applicants/$applicantId/index.tsx

```diff
@@ -117,6 +117,21 @@ export default function Applicant() {
     }
   }, [navigation]);
 
+const appliedIdProjects = applicant.appliedProjectsId?.split(',');
+const appliedNameProjects = applicant.appliedProjects?.split(',');
```

Comment:
why did you use split?

Reply:
To divide the string by "," and convert them into elements of an array. and then save it in "projectMap".